### PR TITLE
docs(core): reword readme

### PR DESCRIPTION
**Pull request by tris0laris**

### CODE REVIEW

**Comment by questdb-butler** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Please update the PR title to match this format:
`type(subType): description`

Where `type` is one of:
`feat`, `fix`, `chore`, `docs`, `style`, `refactor`, `perf`, `test`, `ci`, `revert`

And: `subType` is one of:
`build`, `sql`, `log`, `mig`, `core`, `ilp`, `pgwire`, `http`, `conf`, `ui`, `wal`

For Example:

```
perf(sql): improve pattern matching performance for SELECT sub-queries
```
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 3d53dce758120072bd81427421dc94f469756047
</p>

File: README.md [hunk 1]
```diff
@@ -19,19 +19,24 @@ English | [简体中文](./i18n/README.zh-cn.md) | [繁體中文](./i18n/README.
 
 # QuestDB
 
-QuestDB is an open-source [time-series database](https://questdb.io/glossary/time-series-database/) for high throughput
+QuestDB is an open-source time-series database for high throughput
 ingestion and fast SQL queries with operational simplicity.
 
-QuestDB is well-suited for financial market data, IoT sensor data, application metrics, ad-tech, real-time dashboards, and fast analytics.
+QuestDB is well-suited for financial market data, IoT sensor data, ad-tech and real-time dashboards. It shines for datasets with [high cardinality](https://questdb.io/glossary/high-cardinality/)
+and is a drop in replacement of InfluxDB Line Protocol.
```
Comment (goodroot) [inline README.md:26]:
```suggestion
and is a drop in replacement for InfluxDB via support for the InfluxDB Line Protocol.
```

File: README.md [hunk 2]
```diff
@@ -19,19 +19,24 @@ English | [简体中文](./i18n/README.zh-cn.md) | [繁體中文](./i18n/README.
 
 # QuestDB
 
-QuestDB is an open-source [time-series database](https://questdb.io/glossary/time-series-database/) for high throughput
+QuestDB is an open-source time-series database for high throughput
 ingestion and fast SQL queries with operational simplicity.
 
-QuestDB is well-suited for financial market data, IoT sensor data, application metrics, ad-tech, real-time dashboards, and fast analytics.
+QuestDB is well-suited for financial market data, IoT sensor data, ad-tech and real-time dashboards. It shines for datasets with [high cardinality](https://questdb.io/glossary/high-cardinality/)
+and is a drop in replacement for InfluxDB via support for the InfluxDB Line Protocol.
```
Comment (bluestreak01) [inline README.md:26]:
```suggestion
and is a drop-in replacement for InfluxDB via support for the InfluxDB Line Protocol.
```

File: README.md [hunk 3]
```diff
@@ -19,19 +19,24 @@ English | [简体中文](./i18n/README.zh-cn.md) | [繁體中文](./i18n/README.
...
+and is a drop in replacement of InfluxDB Line Protocol.
 
 QuestDB implements ANSI SQL with native time-series SQL extensions. These SQL extensions make it simple to filter and downsample data, 
-or correlate data from multiple sources using relational and time-series joins. We achieve high performance by adopting a
-column-oriented storage model, parallelized vector execution, SIMD instructions, and low-latency techniques. The entire
-codebase is built from the ground up in Java and C++, with no dependencies and zero garbage collection.
+or correlate data from multiple sources using relational and time-series joins. 
+
+We achieve high performance by adopting a column-oriented storage model, parallelized vector execution, SIMD instructions, and low-latency techniques. 
+The entire codebase is built from the ground up in Java, C++ and Rust (Enterprise) with no dependencies and zero garbage collection.
```
Comment (oliver-daniel) [inline README.md:32]: This kind of makes it sound like the codebase is written in an Enterprise edition of Rust or something. Maybe we could use [footnote syntax](https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/)?

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bluestreak01) [reply README.md]: this conflicted with Kellen's suggeseted but i updated "drop-in" thanks!

File: README.md (hunk 3 above)
Comment (tris0laris) [reply README.md:32]: Yes true, its a bit confusing I agree. Footnote syntax works or alternatively we can remove (enterprise) and link https://questdb.io/blog/leveraging-rust-in-our-high-performance-java-database/ to Rust

Comment (bluestreak01) [reply README.md]: addressed here https://github.com/questdb/questdb/pull/4188/commits/103738de4f35c75ba0ac6d10a7a9a5c5d38c18e5

**Review by bluestreak01 — APPROVED** [review]

**Pull request merged**